iot/api-client/manager/src/main/java/com/example/cloud/iot/examples/DeviceRegistryExample.java

```diff
@@ -884,39 +883,14 @@ public static void unbindDeviceFromGateway(
             .execute();
 
     System.out.println(String.format("Device unbound: %s", response.toPrettyString()));
-    // [END unbind_device_from_gateway]
-  }
-
-  public static void attachDeviceToGateway(MqttClient client, String deviceId)
```

Comment:
Why are you removing this from this file?

Reply:
I moved that into MqttExample, i thought attachDevice related to mqtt

Reply:
I have noticed in python sample, attach, detach methods are in mqtt example instead of manager.py

Do you think it is good idea to move those into MqttExample? or keep in DeviceRegistryManager?

Reply:
This makes sense, we'll just need to remember to update the docs too.